<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Memoire</class>
 <widget class="QMainWindow" name="Memoire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Memoire</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QPushButton" name="btnRestitution">
      <property name="text">
       <string>Cacher</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btnNouveau">
      <property name="text">
       <string>Nouveau jeu de mots</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QListWidget" name="lstLearned"/>
    </item>
    <item row="3" column="0">
     <widget class="QListWidget" name="lstToLearn"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lblChrono">
      <property name="text">
       <string>Temps écoulé.</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="lblResultats">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="btnCorrection">
      <property name="text">
       <string>Correction</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="bxDifficulte"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Difficulté :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" rowspan="2">
     <widget class="QPushButton" name="btnNext">
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="btnPerfs">
      <property name="text">
       <string>Performances</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>543</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fic&amp;hier</string>
    </property>
    <addaction name="actionImporter_des_items"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImporter_des_items">
   <property name="text">
    <string>&amp;Importer des items...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
